call-for-champions.md

```diff
@@ -0,0 +1,97 @@
+# Call for Turing Reproducible Research Champions
+**Application deadline:** Friday 23rd February 2018
+
+## What is a Reproducible Research Champion?
```

Comment:
We don't define reproducible research...should go up near the top somewhere.

Reply:
Added section in commit 36ccd3f.